stripe/functions/index.js

```diff
@@ -63,6 +63,22 @@ exports.createStripeCustomer = functions.auth.user().onCreate(event => {
   });
 });
 
+// Add a payment source (card) for a user by writing a stripe payment source token to Realtime database
+exports.addPaymentSource = functions.database.ref('/stripe_customers/{userId}/cards/{pushId}/token').onWrite(event => {
```

Comment:
now that you're storing `cards/**/*` under `stripe_customers/*` you'll need to change where the stripe id for the customer is stored, as currently that's key/value